src/server/src/districts/services/topology.service.ts

```diff
@@ -82,14 +101,18 @@ export class TopologyService {
         ]);
 
         this.logger.debug(`downloaded data for s3URI ${s3URI}`);
-        return new GeoUnitTopology(
+        const geoUnitTopology = new GeoUnitTopology(
           topology,
           { groups: geoLevelHierarchy.slice().reverse() },
           staticMetadata,
           demographics,
           voting,
           geoLevels
         );
+        this.logger.debug(`finished calculating computed data for s3URI ${s3URI}`);
```

Comment:
Note to self: remove this
```suggestion
        this.logger.debug(`finished calculating computed data for s3URI ${s3URI}`);
```